Makefile

```diff
@@ -1,4 +1,51 @@
-include .mm/magic
-include .mm/bin
-BIN_PKGMODULES=gegl-0.3
+PREFIX=$(shell echo `pwd`)
+PREFIX_DEP=$(PREFIX)/build/imgflo-dependencies/install
+VERSION=$(shell echo `git describe --tags`)
+CFLAGS += -Wall -Wextra
+CFLAGS += -Ofast -g -rdynamic
+CFLAGS += -fPIC
+CFLAGS += -fomit-frame-pointer
+CFLAGS += -funroll-loops -fforce-addr -ftracer -fpeel-loops -fmerge-all-constants
```

Comment:
Nevermind, I see they were in the original Makefiles. I guess we keep them then.